.gitignore

```diff
@@ -1,6 +1,7 @@
 .idea
-/ethereum/.data/
-/bitcoin/.data/
+/docker/bitcoin/.data/
+/docker/ethereum/.data/
```

Comment:
This should hold ethereum explorer cache, right? I'm asking because I don't have this folder, despite successfully running the docker container (likely related to seeing `b-explorer  | Error trying to update latest Transactions cache.` in the container logs).

Reply:
The transaction cache stuff is expected - i'll call that out in the PR description

`./docker/ethereum/.data/` is where we're storing ethereum chain data. `docker-compose.yml` is handling the volume work for us:

```yml
  geth:
    container_name: geth
    build:
      context: .
      dockerfile: Dockerfile.geth
    volumes:
      - ./docker/geth/.data:/ethereum/data
```